superset/charts/post_processing.py

```diff
@@ -86,7 +87,8 @@ def pivot_df(  # pylint: disable=too-many-locals, too-many-arguments, too-many-s
     # pivot data; we'll compute totals and subtotals later
     if rows or columns:
         # pivoting with null values will create an empty df
-        df = df.fillna("NULL")
```

Comment:
mmmh, seems the frontend should be doing this ...

Reply:
@mistercrunch I'm not sure I got this..

Reply:
@mistercrunch the problem here is that we do the pivot in Pandas (for reports and CSV download), and it will fail if the dataframe has `NaN`s.